src/Pact/Types/Command.hs

```diff
@@ -138,22 +144,19 @@ mkCommand creds meta nonce nid rpc = mkCommand' creds encodedPayload
     encodedPayload = J.encodeStrict $ toLegacyJsonViaEncode payload
     payload = Payload rpc nonce meta (keyPairsToSigners creds) nid
 
-keyPairToSigner :: SomeKeyPair -> [SigCapability] -> Signer
+keyPairToSigner :: Ed25519KeyPair -> [SigCapability] -> Signer
 keyPairToSigner cred caps = Signer scheme pub addr caps
-      where scheme = case kpToPPKScheme cred of
-              ED25519 -> Nothing
-              s -> Just s
-            pub = toB16Text $ getPublic cred
-            addr = case scheme of
-              Nothing -> Nothing
-              Just {} -> Just $ toB16Text $ formatPublicKey cred
+      where
+        scheme = Just ED25519
```

Comment:
this is actually a semantic difference.

Reply:
Good catch! I updated it to behave the way it used to for ed25519 keys: `scheme = Nothing`.